src/display/api.js

```diff
@@ -153,6 +159,12 @@ const DefaultStandardFontDataFactory =
  *   when reading the standard font files. Providing a custom factory is useful
  *   for environments without Fetch API or `XMLHttpRequest` support, such as
  *   Node.js. The default value is {DOMStandardFontDataFactory}.
+ * @property {string} [openjpegWasmUrl] - The URL where the openjpeg wasm file
+ * is located.
```

Comment:
This parameter should be consistent with the other URL ones above.
```suggestion
 *   is located. Include the trailing slash.
```